typescript/packages/common-memory/interface.ts

```diff
@@ -229,6 +229,12 @@ export interface QueryError extends Error {
   selector: Selector & { in: ReplicaID };
 }
 
+export interface ListError extends Error {
+  name: "ListError";
+  cause: SystemError;
+  query: { the: The, in: ReplicaID };
```

Comment:
Not sure if `query` is really the right name for this. Maybe it's a `selector` instead?